### ISSUE TITLE
Adding Activity Template data model into gatsby

### ISSUE THREAD

**Issue by wwwwwlwwwww** [distributeaid/distributeaid.org#583]
NOTE: #571 needs to be completed before this should be done.

The Activity data model is available on Forestry.  We need to load it into Gatsby, so we can consume that data to make What We Do pages.

It has the following fields:
* title
* photo*
* desc
* challenge
* explanation
* benefits
* regions (repeatable field group so there can be multiple regions in here)
  * region
 
*This is the field that is waiting on #571.

**Comment by jtfairbank** [comment]
Duplicate of #616
